docs/install.md

```diff
@@ -0,0 +1,203 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard.
+
+## Pre-requisites
+
+[Kubernetes](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later if you want to use a version of Tekton Pipelines newer than v0.10.1.
+
+[Tekton Pipelines](https://github.com/tektoncd/pipeline) must be installed in order to use the Tekton Dashboard. Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md). 
+
+### Which version should I use?
+
+It is strongly recommended to use the *v0.6.1.4* release for Tekton Pipelines v0.11.x and v0.12.x. This is a critically important security release and can display components from Triggers 0.4.
+
+The following versions should be for development or isolated usage only and are *deprecated*. It is strongly advised to pick up the latest releases as soon as possible.
+
+- Use the v0.6.0 release for Tekton Pipelines v0.11.x (can display components from Tekton Triggers 0.3.x)
+- Use the v0.5.3 release for Tekton Pipelines v0.10.x (can display components from Tekton Triggers 0.3.x)
+- Use the v0.5.2 release for Tekton Pipelines v0.10.x (can display components from Tekton Triggers 0.2.x)
+- Use the v0.5.0 release for Tekton Pipelines v0.10.x (can display components from Tekton Triggers 0.1 and all versions from now on have read-only install modes)
+- Use the v0.4.1 release for Tekton Pipelines v0.8.0 (can display components from Tekton Triggers 0.1)
+- Use the v0.2.1 release for Tekton Pipelines v0.7.0
+- Use the v0.1.1 release for Tekton Pipelines v0.5.2
+
+## Install Dashboard
+
+### Installing the latest release
+
+1. Run the
+   [`kubectl apply`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply)
+   command to install the [Tekton Dashboard](https://github.com/tektoncd/dashboard)
+   and its dependencies:
+
+   ```bash
+   kubectl apply --filename https://github.com/tektoncd/dashboard/releases/download/v0.6.1.4/tekton-dashboard-release.yaml
```

Comment:
@a-roberts I can't remember, is there a reason we don't use `latest` here anymore?